config/content/About.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable max-len */
+export default [
```

Comment:
Change this export default to export const as other things would also be exported for the about file. Also the file name, change the capital to lowercase. As per the convention, only component names are written in Capital case.